OracleDatabase/RAC/OracleRealApplicationClusters/dockerfiles/19.3.0/AddNode.sh

```diff
@@ -11,14 +11,13 @@
 #
 # DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
 #
+# shellcheck disable=SC2181,SC2016,SC1091,SC2086
```

Comment:
None of these should be disabled as they're all pretty straightforward to fix. Please fix the actual issues instead of suppressing them.